rules/windows/file_event/win_susp_multiple_files_renamed.yml

```diff
@@ -0,0 +1,27 @@
+title: Suspicious Multiple File Rename Occurred
+id: 97919310-06a7-482c-9639-92b67ed63cf8
+author: Vasiliy Burov, oscd.community
+date: 2020/10/16
+description: Detects multiple file rename events occurrence within a specified period of time by a same user. These events may signalize about ransomware activity.
+status: experimental
+references:
+    - https://www.manageengine.com/data-security/how-to/how-to-detect-ransomware-attacks.html
+tags:
+    - attack.impact
+    - attack.t1486
+logsource:
+    product: windows
+    service: security
+    definition: 'Requirements: Audit Policy : Policies/Windows Settings/Security Settings/Local Policies/Audit Policy/Audit object access, Policies/Windows Settings/Security Settings/Advanced Audit Policy Configuration/Object Access'
+detection:
+    selection:
+        EventID: 4663
+        ObjectType: 'File'
+        SubjectLogonId: '*'
```

Comment:
Hello @vburov! Thank you for your continuous contribution!
Could you please tell me if the `SubjectLogonId: '*'` is really required here?
We are trying to avoid the usage of the `*` character in the detection section. 
Is it possible to add a filter with the empty `SubjectLogonId` field? 
That would solve the issue. 

Thanks!

Reply:
Hi, Daniil!
Thanks for your note. I changed the '*' value to 'not null' for 'SubjectLogonId' parameter.

Reply:
Hello @vburov! It seems that "not null" logic [doesn't work anymore](https://github.com/Neo23x0/sigma/wiki/Specification#special-field-values). I've converted your rule using splank backend and here is the result:

```
(source="WinEventLog:Security" EventCode="4663" ObjectType="File" SubjectLogonId="not null" AccessList="%%1537" Keywords="0x8020000000000000") | eventstats count as val by SubjectLogonId| search val > 20
```

Here is a working example from wiki:

> To get an expression that say not null you have to create another selection and negate it in the condition.
>
>Example:
>
>```
>detection:
>   selection:
>      EventID: 4738
>   filter:
>      PasswordLastSet: null
>condition:
>   selection and not filter
>```

Could you please adjust your rule following the example? Thank you!

Reply:
Hi, Daniil! Thanks for tests! I decided to completely remove this condition. I made a small research and seems that this field cannot be empty.

Reply:
Awesome! Thank you